linux/debian/control

```diff
@@ -37,8 +37,8 @@ Build-Depends:
  metacity,
  gawk,
 Standards-Version: 4.6.1
-Vcs-Git: https://github.com/keymanapp/keyman.git [linux/debian]
-Vcs-Browser: https://github.com/keymanapp/keyman/tree/master/linux/debian
+Vcs-Git: https://github.com/keymanapp/keyman.git -b stable-16.0 [linux/debian]
+Vcs-Browser: https://github.com/keymanapp/keyman/tree/stable-16.0/linux/debian
```

Comment:
Should these stay master?

Reply:
I think it doesn't matter. We don't use this information anywhere, it's just used on Debian with their packaging infrastructure. Keeping the file content the same on stable and master simplifies cherry-picking IMO.